src/endpoints/Home.php

```diff
@@ -10,7 +10,7 @@ class Home extends Route
 {
     public function __invoke(Request $request, Response $response)
     {
-        $response = $response->withRedirect('/admin');
+        $response = $response->withRedirect('./admin/');
```

Comment:
This makes sure the redirect works well when the API is nested in another folder, eg `/api/admin`